src/features/RodeoDashboard/RodeoForms/CreateEventModal.tsx

```diff
@@ -35,9 +35,17 @@ const CreateEventModal: React.FC<Props> = ({defaultValues, onSubmit, onClose}) =
       <DialogTitle>Create event</DialogTitle>
       <DialogContent>
         <DialogContentText>
```

Comment:
Probably easiest way to add padding for now is here, by giving this `sx={{paddingBottom: 3}}` or something